src/strategies/erc3525-vesting-voucher/index.ts

```diff
@@ -0,0 +1,40 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { claimCoefficient, maturitiesCoefficient } from './utils';
+
+export const author = 'buchaoqun';
+export const version = '0.1.0';
+
+const abi = [
+  'function getSnapshot(uint256 tokenId_) view returns (uint8 claimType_, uint64 term_, uint256 vestingAmount_, uint256 principal_, uint64[] maturities_, uint32[] percentages_, uint256 availableWithdrawAmount_, string originalInvestor_, bool isValid_)'
+];
+
+export async function strategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+) {
+    const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+    const snapshotData = await multicall(
+        network,
+        provider,
+        abi,
+        addresses.map((address: any) => [
+            options.address,
+            'getSnapshot',
+            [options.tokenId]
+        ]),
```

Comment:
@buchaoqun Looks like here same thing is being sent into multicall for each voter address, please look into it :)